decisions/mnm_society_decisions.txt

```diff
@@ -2756,64 +2751,106 @@ targeted_decisions = {
 		}
 		
 		allow = {
+			# Warcraft
+			# If you're not in society, you can abduct courtiers only
+			OR = {
+				is_ruler = no
+				FROM = { is_in_society = yes }
+			}
+			
 			in_revolt = no
 			NOT = { war_with = FROM }
 			NOT = { same_society_as = FROM }
 			prisoner = no
-			OR = {
-				AND = {
-					is_landed = no
-					FROM = {
-						has_society_currency_tiny_trigger = yes
+			
+			# Warcraft
+			trigger_if = {
```

Comment:
I didn't get it. Isn't conditions are the same?

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/75932529-a8df6f80-5e88-11ea-8b56-de7d9475ebaa.png)

</details>

Reply:
The first if is shown if you have society currency. If you don't have, vampires see their own condition, non-vampires see the else trigger.

Reply:
Add comment about it, pls

Reply:
Added in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/commit/b8fc8f8ef0aa4893f016a1018991f26406def8bd.